### PR TITLE
Translate Juice to fr

### PR DESCRIPTION
**Pull request by Flayor**
Hi,

I Have translate this app to FR

### CODE REVIEW

**Comment by brianmichel** [comment]
Wow this looks incredible! I believe that Apple recommends that the keys for all localizable strings actually be the project base language variant of the string so if a user has their computer in an unsupported locale they will at least see well constructed strings. So instead of 

```
CHARGED="chargés";
```
I think we should have it say

```

```
"charged"="chargés";
```

and the same thing for the english keys (even though it might look a bit silly 😊) this way if a user has their laptop in say German they will at least see `charged` instead of `CHARGED`

**Comment by Flayor** [comment]
Ok no problem, 

I minus all the keys and finish option :)

Please tell me just how to get your bio in the app :) 

thanks

**Comment by Flayor** [comment]
I forget, i see a bug ,

If you change scale, emoji scale are not in place

Example if i change, and i want again emoji, i do to choice "Dotted Line" :)

**Comment by brianmichel** [comment]
> I forget, i see a bug ,
> 
> If you change scale, emoji scale are not in place
> 
> Example if i change for "Roman Numerals", and i want again emoji, i do to choice "Dotted Line" :)

Interesting, would you be able to file an issue with the steps to reproduce the problem please?

**Comment by brianmichel** [comment]
> Please tell me just how to get your bio in the app :)

Ha! this was a screen i decided not to put in and will be deleting the code since I more easily could just write it in the README, I'll be deleting that code soon though, enjoy the picture of me lol

File: Juice/Classes/ChargeScaleDisplay.swift [hunk 1]
```diff
@@ -18,9 +19,9 @@ protocol ChargeScaleDisplay {
 }
 
 enum SerializableChargeScaleDisplayConstants {
-    static let title = "title"
-    static let detents = "detents"
-    static let defaultDetentString = "default"
+    static let title = NSLocalizedString("title", comment: "title")
```
Comment (brianmichel) [inline Juice/Classes/ChargeScaleDisplay.swift]: These should not be localized as they are part of the plist serialization format.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
 ![](/images/icon-readme.png)
+
+Thanks to brianmichel, i just translate Juice to French people contact me for bug or other translate : contact@flayor.com
```
Comment (brianmichel) [inline README.md]: Can we add a `Contributors` section to the end of the read me please? Also, I'm happy to acknowledge you for the work you've done on the project, but I do not want to have the contact email address as part of this projects README, but feel free to link to your github/twitter profile.

File: fr.lproj/Localizable.strings [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+"Version Information"="Information de la version";
+"Power Adapter"="adaptateur secteur";
+"Battery"="Batterie";
+"Unknown"="Inconnue";
+"Off Line"="Hors Ligne";
+"External Power"="Alimentation Externe";
+"General"="Général";
+"General setting"="Réglage général";
+"Credits"="Crédits";
+//"Scales Found"="Skin trouvé";
```
Comment (brianmichel) [inline fr.lproj/Localizable.strings]: Can this be deleted if it's not needed? Or should it not be commented out?

File: fr.lproj/Localizable.strings [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+"Error scanning application support directory"="Error scanning application support directory";
```
Comment (brianmichel) [inline fr.lproj/Localizable.strings]: This string does not seem to be translated.

File: fr.lproj/Localizable.strings [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+"title"="titre";
+"default"="Défaut";
+"detents"="detents";
+"Error serializing scale"="Error serializing scale";
```
Comment (brianmichel) [inline fr.lproj/Localizable.strings]: This string does not seem to be translated.

**Comment by Flayor** [comment]
I have recreate an other project merge confilt, i open an other pull request.

For this bug ... this is strange i don't have this problem after reboot & maj Puice

**Pull request closed without merging**